packages/marshal/test/test-encodePassable.js

```diff
@@ -67,7 +67,7 @@ const encodePassableInternal = makeEncodePassable({
   encodeError: er => encodeThing('!', er),
 });
 
-const encodePassable = passable => {
+export const encodePassable = passable => {
```

Comment:
I do not feel strongly about this, but I think a utility for multiple tests should be ejected to a non `test-` prefix module. I suspect that Ava might do something weird if a `test-` imports another `test-`, or does something extraordinary to avoid doing something weird.

Reply:
yes, if a imports b, then `ava a` will run all of b too.

> modules should not export powerful objects
> Note that tests don't export anything.
-- https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline

Reply:
Good suggestion. Nice to have the common stuff factored out. Done.